src/dialog.c

```diff
@@ -422,3 +477,13 @@ bool dialog_is_active(Dialog *d) {
 void dialog_preload(void) {
 	preload_resource(RES_SHADER_PROGRAM, "text_dialog", RESF_DEFAULT);
 }
+
+void dialog_draw_title(Dialog *dialog, DialogActor *actor, char *name, char *title) {
+	dialog->title.name = name;
+	dialog->title.text = title;
+	dialog->title.active = true;
+	dialog->title.timeout = 360;
+
+	log_debug("Show Title: %s - %s", name, title);
+
+}
```

Comment:
This function does not draw anything, it only sets up state. It should be renamed `dialog_show_title` or similar.

The `draw` verb is generally reserved for functions that do rendering and do not change state. Any existing counter-example to this convention should be considered a bug.